rasa_core/server.py

```diff
@@ -146,25 +146,26 @@ def _create_agent(
         return None
 
 
-def create_app(model_directory,
-               interpreter=None,
-               loglevel="INFO",
-               logfile="rasa_core.log",
-               cors_origins=None,
-               action_factory=None,
-               auth_token=None,
-               tracker_store=None
+def create_app(model_directory,           # type: Text
+               interpreter=None,          # type: Optional
+                                        # [Union(text,NL interpreter)]
```

Comment:
this isn't a proper type hint, should be `Union[Text, NLI, None]` with a

```
if typing.TYPE_CHECKING:
    from rasa_core.interpreter import NaturalLanguageInterpreter as NLI
```
at the top (after the imports).

additionally you need to add 
```
from typing import Union
import typing
```
to the imports

Reply:
please also don't align the type hints but rather put them two spaces after the argument.